### PR TITLE
Refactor settings for unit test cases

### PR DESCRIPTION
**Pull request by ranvijayj**
- Refactored base.py for units tests settings
- Commented disable migrations for now. Might need it later
and unused imports
- Removed __init__.py from root directory

### CODE REVIEW

File: chatbot_ner/settings.py [hunk 1]
```diff
@@ -60,35 +60,35 @@
 # Database
 # https://docs.djangoproject.com/en/1.6/ref/settings/#databases
 
+# FOR TEST CASES - COMMON SETTINGS FOR ALL ENVIRONMENTS
+class DisableMigrations(object):
```
Comment (srajdev) [inline chatbot_ner/settings.py]: E302 expected 2 blank lines, found 1

**Comment by srajdev** [comment]
:+1: No lint errors found.

File: ner_v1/detectors/textual/name/tests/test_name_detection.py [hunk 2]
```diff
@@ -1,35 +1,29 @@
...
             'expected_value': [],
         }
-        for language, message, first_name, middle_name, last_name, original_entity in zip(self.data['language'],
-                                                                                          self.data['message'],
-                                                                                          self.data['first_name'],
-                                                                                          self.data['middle_name'],
-                                                                                          self.data['last_name'],
-                                                                                          self.data
-                                                                                          ['original_entities']):
+        zipped = zip(self.data['language'], self.data['message'], self.data['first_name'],
```
Comment (srajdev) [inline ner_v1/detectors/textual/name/tests/test_name_detection.py]: W1637 zip built-in referenced when not iterating

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by prathamghadge** [comment]
retest this please

**Pull request merged**